### ISSUE TITLE
Changes to ChatGPT web site causing pyChatGPT to no longer work?

### ISSUE THREAD

**Issue by trzy** [terry3041/pyChatGPT#50]
Hi, 

Thanks for this incredible project :) I used it as recently as earlier today but it seems that something has changed in the page layout and pyChatGPT bails with:

```
Traceback (most recent call last):
  File "C:\Users\Bart\anaconda3\envs\chatgpt\lib\runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\Bart\anaconda3\envs\chatgpt\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "C:\projects\ChatGPT\chatgpt.py", line 183, in <module>
    loop.run_until_complete(asyncio.gather(*tasks))
  File "C:\Users\Bart\anaconda3\envs\chatgpt\lib\asyncio\base_events.py", line 649, in run_until_complete
    return future.result()
  File "C:\projects\ChatGPT\chatgpt.py", line 40, in run
    self._chat_gpt.reset_conversation()
  File "C:\Users\Bart\anaconda3\envs\chatgpt\lib\site-packages\pyChatGPT\pyChatGPT.py", line 418, in reset_conversation
    self.driver.find_element(By.LINK_TEXT, 'New Chat').click()
  File "C:\Users\Bart\anaconda3\envs\chatgpt\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 861, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "C:\Users\Bart\anaconda3\envs\chatgpt\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 444, in execute
    self.error_handler.check_response(response)
  File "C:\Users\Bart\anaconda3\envs\chatgpt\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 249, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"link text","selector":"New Chat"}
```

Has anyone else encountered this? I am using the session token API.

Thank you,

Bart

**Comment by DevRoss** [comment]
You can refer to the PR below to solve this issue:
[PR52](https://github.com/terry3041/pyChatGPT/pull/52)
[PR42](https://github.com/terry3041/pyChatGPT/pull/42)

or just replace "New Chat" with "New chat" in code
```python
# before
self.driver.find_element(By.LINK_TEXT, 'New Chat').click()
# after
self.driver.find_element(By.LINK_TEXT, 'New chat').click()
```

**Comment by terry3041** [comment]
Should be fixed in v0.3.9.5.
